### DISCUSSION TITLE
Ist es möglich über Jarvis einen Datenpunkt mit on/off zu steuern ?

### DISCUSSION THREAD

**Discussion by tardar** [Zefau/ioBroker.jarvis#2164] [General]
Hi zusammen,

ich möchte gerne einen Datenpunkt, den ich manuell angeegt habe in Jarvs mit einem ON/OFF Button steuern.
Hat das schon jemand bewerkstelligt?
Ich bin nicht so fit in JSON und würde mich riesig über Hilfe freuen :)

Das ist meine aktuelle Config für den Button, der Eintrag wird mir auf der Jarvis aber noch nicht angezeigt:
<img width="970" alt="iobroker_jarvis_datenpunkt_mit_jarvis_steuern" src="https://user-images.githubusercontent.com/40645000/212469161-ecfb34a6-89a0-4394-8ee8-0d53762ad16c.png">

Ganz lieben Dank und beste Grüße
Tardar

**Comment by mcuiobroker** [comment]
Das ist kein Datenpunkt, sondern ein Verzeichnis mit dem Punkt.
Also entweder den Punkt entfernen oder dahinter noch etwas ergänzen.
Weiterhin ist es empfehlenswert seine eigenen DPs unter `0_userdata.0` anzulegen.
